api/generator/generator.go

```diff
@@ -275,6 +284,15 @@ func (cg *CertGeneratorHandler) Handle(w http.ResponseWriter, r *http.Request) e
 		},
 	}
 
+	if req.Bundle && cg.bundler != nil {
```

Comment:
Should a message be sent to the user if `req.Bundle` is `true` but no bundler is defined?